### PR TITLE
[wasm64] Fix library_glemu.js under wasm64

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/library_webgl.js [hunk 1]
```diff
@@ -4279,6 +4279,10 @@ function recordGLProcAddressGet(lib) {
   Object.keys(lib).forEach((x) => {
     if (x.startsWith('gl') && !isDecorator(x)) {
       lib['emscripten_' + x] = x;
+      var sig = LibraryManager.library[x + '__sig'];
```
Comment (dschuff) [inline src/library_webgl.js:4282]: how is this part related to the rest of it?

**Review by dschuff — APPROVED** [review]

File: src/library_webgl.js (hunk 1 above)
Comment (sbc100) [reply src/library_webgl.js:4282]: This lack of correct signatures for `emscripten_glXXXX` wrappers was causing the test to fail.
